Jenkinsfile

```diff
@@ -1,30 +1,35 @@
 #!/usr/bin/env groovy
 
 node(label: "jenkins-slave") {
+
   try {
     stage("Checkout") {
-      sh 'echo Checking out code from github'
       checkout scm
     }
     stage("Build") {
-      sh '''
-        echo Starting the build...
-      '''
+      sh 'eval $(cat rc_dev) && make cleanall all'
     }
-    stage("Run") {
-      sh '''
-        echo Starting the containers...
-      '''
+    stage("Lint") {
+      sh 'make lint'
     }
     stage("Test") {
-      sh '''
-        echo Starting the tests...
-      '''
+      parallel (
+        'integration': {
+          sh '.venv/bin/nosetests chsdi/tests/integration/'
+        },
+        'functional': {
+          sh '.venv/bin/nosetests chsdi/tests/functional/'
+        }
+      )
     }
+  }
```

Comment:
No deploy on int or dev targets ?

Reply:
no not possible. We don't have the `deploy` script on this instance